public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx

```diff
@@ -1,19 +1,29 @@
 import React, { ChangeEvent } from 'react';
-import { Button, HorizontalGroup, Input, Label, LoadingPlaceholder, withTheme2 } from '@grafana/ui';
+import {
+  Button,
+  HighlightPart,
+  HorizontalGroup,
+  Input,
+  Label,
+  LoadingPlaceholder,
+  withTheme2,
+  BrowserLabel as LokiLabel,
+} from '@grafana/ui';
 import LokiLanguageProvider from '../language_provider';
 import PromQlLanguageProvider from '../../prometheus/language_provider';
 import { css, cx } from '@emotion/css';
 import store from 'app/core/store';
 import { FixedSizeList } from 'react-window';
-
 import { GrafanaTheme2 } from '@grafana/data';
-import { LokiLabel } from './LokiLabel';
+import { fuzzyMatch } from '../../../../../../packages/grafana-ui/src/slate-plugins/fuzzy';
```

Comment:
I think it'd be better to move fuzzy.ts to grafana-ui/utils, mark it as `@internal` and import here as `import { fuzzyMatch } from '@grafana/ui';`